src/BankSystem/Account.java

```diff
@@ -1,13 +0,0 @@
-package BankSystem;
-
-public class Account extends BankAccount{
-    private String accountType;
-    public void services(){
-        System.out.println("you are an advanced client");
-    }
```

Comment:
In case that this method will always print the same line regardless of the `accountType`, what's the use for `accountType`? and why do we need to pass it in the constructor? 

`Account account=new Account("123456789",5000,"advanced");`